src/core/rewrite.ml

```diff
@@ -84,8 +84,8 @@ let check_builtin : popt -> sym StrMap.t -> string -> sym -> unit
        and x = Bindlib.new_var mkfree "x"
        and y = Bindlib.new_var mkfree "y" in
        let ta = Appl (symb symb_T, Vari a) in
-       let c = [(y, ta); (x, ta); (a, term_U)] in
-       let eq_type = Ctxt.to_prod c term_Prop in
+       let c = [(y, ta, None); (x, ta, None); (a, term_U, None)] in
+       let eq_type, _ = prod c term_Prop in
```

Comment:
Use the `snd` function, or add parentheses around `eq_type, _`.

There are two occurrences of that in this file.